redis/src/cluster_client.rs

```diff
@@ -212,6 +222,13 @@ impl ClusterClientBuilder {
                 )));
             }
 
+            if client_name.is_some() && node.redis.client_name != *client_name {
```

Comment:
I think we said it should be `if client_name != node.redis.client_name {`. rigtht?